nixnet/_session/intf.py

```diff
@@ -8,7 +8,7 @@
 
 from nixnet import _props
 from nixnet import constants
-from nixnet.db import _frame
+from nixnet.database import _frame
```

Comment:
I was surprised at first that we didn't need to update the rst file for this but we haven't created it yet

https://github.com/ni/nixnet-python/tree/master/docs/api_reference

Reply:
I actually started creating the rst and that's exactly what prompted me to want this done before I start churning out database documentation.